dbus/subscription.go

```diff
@@ -169,6 +169,13 @@ type SubStateUpdate struct {
 // is full, it attempts to write an error to errCh; if errCh is full, the error
 // passes silently.
 func (c *Conn) SetSubStateSubscriber(updateCh chan<- *SubStateUpdate, errCh chan<- error) {
+	if c == nil {
+		select {
+		case errCh <- errors.New("nil receiver"):
```

Comment:
I'm not familiar with this codebase, but is there a chance of a situation where writing to `errCh` will block indefinitely? Returning without propagating the error up in any way makes me nervous. Maybe log something at the very least in the `default` case?

Reply:
`if errCh is full, the error passes silently` says the public docstring and other full errCh are ignored too. I understand your feeling though, and I'm trying to cleanup a bit the code. Should I go ahead and sprinkle `log.Printf` around?